waterloop-site/package.json

```diff
@@ -6,6 +6,7 @@
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.60",
+    "@styled-icons/evil": "^10.34.0",
```

Comment:
To stay inline with what @williamhpark was saying with your other PR, is there a way to incorporate libraries we already used (e.g material-ui) instead of installing another dependency?

Reply:
Yep you should be able to use this: https://fonts.google.com/icons?icon.query=close with `@material-ui/icons`